zbus/src/connection/builder.rs

```diff
@@ -77,11 +77,19 @@ assert_impl_all!(Builder<'_>: Send, Sync, Unpin);
 
 impl<'a> Builder<'a> {
     /// Create a builder for the session/user message bus connection.
+    ///
+    /// # Errors
+    ///
+    /// Returns an error variant if parsing the session bus address fails.
```

Comment:
nit: We should either remove "variant" or explicitly mention the variant name.

Reply:
I don't think this is terribly useful, but I've mentioned the variant explicitly anyway.